.github/buildomat/jobs/build-and-test.sh

```diff
@@ -14,6 +14,18 @@ set -o xtrace
 cargo --version
```

Comment:
The change here - and in the ".github/workflows/rust.yml" file - just make sure `cockroach` is downloaded and on the PATH *before* we `cargo build --all-targets`, since we're now using it in a build script.

Reply:
Could you update the README to reflect this?  Basically this part:

> CockroachDB v21.1.10.
> The test suite expects to be able to start a single-node CockroachDB cluster using the cockroach executable on your PATH.

I think that should now say "the build and test suite expect..."